capnpc-go/capnpc-go.go

```diff
@@ -751,6 +754,18 @@ func makeTypeRef(t schema.Type, rel *node, nodes nodeMap) (typeRef, error) {
 		case schema.Type_Which_anyPointer, schema.Type_Which_list, schema.Type_Which_interface:
 			return typeRef{name: "PointerList", newfunc: "NewPointerList", imp: capnpImportSpec}, nil
 		}
+	case schema.Type_Which_anyPointer:
+		switch t.AnyPointer().Which() {
+		case schema.Type_anyPointer_Which_unconstrained:
+
+			switch t.AnyPointer().Unconstrained().Which() {
+			case schema.Type_anyPointer_unconstrained_Which_capability:
+				return typeRef{name: "Client", imp: capnpImportSpec}, nil
```

Comment:
won't this return `Client`? I think we want `*Client`.

Edit: Ah, I see, you worked around this by putting the `*` in the template. It feels like a bit of a hack...

Reply:
Yeah, it's not _great_.  I looked for a means of specifying a pointer in `capnpImportSpec`, but didn't see anything obvious.  Notably, `importSpec{path: capnpImport, name: "*capnp"}` produces an error.

I'm resolving this, but I'm happy to make improvements if you have any suggestions.

Reply:
I'm fine leaving this as-is though I don't love it.